pom.xml

```diff
@@ -33,7 +33,7 @@
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-chrome-driver</artifactId>
             <scope>test</scope>
-            <version>${chrome.driver.version}</version>
+            <version>${selenium.java.version}</version>       
```

Comment:
change version to needed in chrome.driver.version

Reply:
fixed